src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -31,14 +34,17 @@
     private final String telegram;
     private final String matricNumber;
     private final List<JsonAdaptedTag> tagged = new ArrayList<>();
+    private final List<JsonAdaptedAttendance> attendances = new ArrayList<>();
 
     /**
      * Constructs a {@code JsonAdaptedPerson} with the given person details.
      */
     @JsonCreator
     public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
-            @JsonProperty("email") String email, @JsonProperty("telegram") String telegram,
-            @JsonProperty("matricNumber") String matricNumber, @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+                             @JsonProperty("email") String email, @JsonProperty("telegram") String telegram,
+                             @JsonProperty("matricNumber") String matricNumber,
+                             @JsonProperty("tagged") List<JsonAdaptedTag> tagged,
```

Comment:
Minor naming convention - Shouldn't this be **tags**  since this is a collection

Reply:
Good point! Seems like that was from AB3, but I'll fix it either way.

Reply:
Fixed in https://github.com/AY2021S1-CS2103T-F11-1/tp/pull/73/commits/0232a53dbc5ea8e2ae8158208d2e23ebe3c7c877